### DISCUSSION TITLE
Icecast ssl error alfter renew ssl err_ssl_version_or_cipher_mismatch

### DISCUSSION THREAD

**Discussion by caf2022** [AzuraCast/AzuraCast#6938] [General]
I'm getting error with icecast after renew ssl, I think that is a incompatible TLS/cipher, when try to access the icecast page, show me the message "err_ssl_version_or_cipher_mismatch", but the SSL works fine with shoutcast, and was working fine before renew.

**Comment by BusterNeece** [comment]
This _should_ be fixed in the newest release versions of AzuraCast; I'd recommend updating and confirming that it's fixed, and if it isn't then feel free to create a new issue (this was created as a discussion, which doesn't let us tag and triage it properly).
